action_list:

  number_of_target: 7

  # starting position
  action_0:
    action_type: pose_action
    translation: [ 0.315, 0.10, 0.5 ]
    orientation: [ 0.0, 0.0, 0.0 ]
    duration: 5.0

  # open the gripper
  action_1:
    action_type: gripper_action
    position: 0.1
    max_effort: 1

  # move to pre-grasp position
  action_2:
    action_type: pose_action
    translation: [ 0.315, 0.10, 0.43 ]
    orientation: [ 0.0, 0.0, 0.0 ]
    duration: 3.0

  # Grab the cube
  action_3:
    action_type: gripper_action
    position: 0.61
    max_effort: 0.8

  # move the cube to the cup position
  action_4:
    action_type: pose_action
    translation: [ 0.315, 0.0, 0.55 ]
    orientation: [ 0.0, 0.0, 0.0 ]
    duration: 6.0

  action_5:
    action_type: pose_action
    translation: [ 0.43, 0.11, 0.6 ]
    orientation: [ 0.0, 0.0, 0.0 ]
    duration: 5.0

  # open the gripper to drop the cube into the cup
  action_6:
    action_type: gripper_action
    position: 0.1
    max_effort: 1
